Adding links for exploring ivvv in MyBinder [WIP]

kmader opened the pull request:
adding the code and links for playing around with the module in mybinder

kmader committed:
adding binder settings

adding binder environment

kmader committed:
fixing environment name

kmader committed:
Create postBuild

adding install script

kmader committed:
Update README.md

adding binder links

kmader committed:
Add files via upload

adding a 3d foam example

kmader committed:
Update postBuild

adding command to extract the dataset

kmader committed:
Update README.md

adding link to 3d notebook moving other links to under development

dmt-aics commented on README.md:
```diff
@@ -1,7 +1,9 @@
 # ivvv
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/kmader/ivvv/patch-2)
```
In order to merge this into imaging-tools:master shouldn't all the gh/kmader lines be changed to gh/imaging-tools ?

dmt-aics commented on README.md:
```diff
@@ -18,6 +20,17 @@ Standard scientific python toolkit.
+
+### Demo Notebooks
+
+- [Foam 3D](https://mybinder.org/v2/gh/kmader/ivvv/patch-2?filepath=examples%2Ffoam_3d.ipynb)
```
Same as above - in order to merge this into imaging-tools:master shouldn't all the gh/kmader lines be changed to gh/imaging-tools ?

kmader replied:
Yes I didn't want to change them because the links then don't work (until the PR is accepted) but I can change them now

kmader committed:
Update README.md

fixing links to point to imaging-tools instead of kmader
old links are 
[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/kmader/ivvv/patch-2)
### Demo Notebooks

- [Foam 3D](https://mybinder.org/v2/gh/kmader/ivvv/patch-2?filepath=examples%2Ffoam_3d.ipynb)

#### Under development

- [Demo](https://mybinder.org/v2/gh/kmader/ivvv/patch-2?filepath=examples%2Fdemo.ipynb)
- [Hello World](https://mybinder.org/v2/gh/kmader/ivvv/patch-2?filepath=examples%2Fhello_world.ipynb)

toloudis commented:
Thank you for the contribution!
